sdcm/cluster_gce.py

```diff
@@ -359,35 +365,16 @@ def _node_setup(self, node):
         node.remoter.run('sudo cat /etc/scylla.d/io.conf')
         node.wait_db_up()
         node.wait_jmx_up()
+        node.remoter.run('sudo yum install -y {}-gdb'.format(node.scylla_pkg()),
+                         verbose=verbose, ignore_status=True)
 
     def wait_for_init(self, node_list=None, verbose=False, timeout=None):
```

Comment:
why don't use @wait_for_init_wrap for this wiat_for_init()?

Reply:
It's called here - via parent wait_for_init call(the next line)

Reply:
OK